### PR TITLE
Feature/c menu

### PR DESCRIPTION
**Pull request by JucanAndreiDaniel**
Add c_menu and c_menu_item data format and data

### CODE REVIEW

**Review by JaXt0r — CHANGES_REQUESTED** [review]
Test failed unfortunately:
image

**Comment by JucanAndreiDaniel** [comment]
> Test failed unfortunately: image

It's weird, for me it doesn't give issues

**Comment by JaXt0r** [comment]
> > Test failed unfortunately: image
> 
> It's weird, for me it doesn't give issues

Can you please check with German game language? It seems to be a language agnostic check which shouldn't be.

**Comment by JucanAndreiDaniel** [comment]
> > > Test failed unfortunately: image
> > 
> > 
> > It's weird, for me it doesn't give issues
> 
> Can you please check with German game language? It seems to be a language agnostic check which shouldn't be.

image
This is with the german version

**Comment by JaXt0r** [comment]
> > > > Test failed unfortunately: image
> > > 
> > > 
> > > It's weird, for me it doesn't give issues
> > 
> > 
> > Can you please check with German game language? It seems to be a language agnostic check which shouldn't be.
> 
> image This is with the german version

Exactly. So the demand for a test is, that it is language independent if possible. (Sure Russian fails as well, but we should have green tests for German and English at least as these are the languages we develop with).

**Comment by JucanAndreiDaniel** [comment]
> > > > > Test failed unfortunately: image
> > > > 
> > > > 
> > > > It's weird, for me it doesn't give issues
> > > 
> > > 
> > > Can you please check with German game language? It seems to be a language agnostic check which shouldn't be.
> > 
> > 
> > image This is with the german version
> 
> Exactly. So the demand for a test is, that it is language independent if possible. (Sure Russian fails as well, but we should have green tests for German and English at least as these are the languages we develop with).

Agree with you, should we merge this PR and have another PR to solve these issues?

**Comment by JaXt0r** [comment]
It was just one test from your new main menu feature. Please fix it with this PR. Others are working already fine for ger and eng which is sufficient imho.

**Comment by JucanAndreiDaniel** [comment]
> It was just one test from your new main menu feature. Please fix it with this PR. Others are working already fine for ger and eng which is sufficient imho.

Ok, now i know what the issue was, i forgot to push to phoenix-shared the c-menu stuff :D
can be merged after https://github.com/GothicKit/phoenix-shared-interface/pull/38

File: PxCs/Interface/PxVm.cs [hunk 1]
```diff
@@ -17,52 +17,112 @@ public enum PxVmInstanceType
...
+            ITEM_KAT_MUN = 1 << 3, // munition (->MultiSlot)
+            ITEM_KAT_ARMOR = 1 << 4, // armor and helmets
+            ITEM_KAT_FOOD = 1 << 5, // food (->MultiSlot)
+            ITEM_KAT_DOCS = 1 << 6, // documents
+            ITEM_KAT_POTIONS = 1 << 7, // potions
+            ITEM_KAT_LIGHT = 1 << 8, // light sources
+            ITEM_KAT_RUNE = 1 << 9, // runes and scrolls
+            ITEM_KAT_MAGIC = 1 << 31, // rings and amulets
+            ITEM_KAT_KEYS = ITEM_KAT_NONE,
+
```
Comment (JaXt0r) [inline PxCs/Interface/PxVm.cs:41]: Please stop your formatter from reformatting all the lines. ;-)

e.g. these lines were intentionally intended this way.

**Review by JaXt0r — APPROVED** [review]
Please change your formatter to "do not be greedy". Example below.
Other than that: Test is working and feature looks good. :-)

**Pull request merged**